content/en/docs/tasks/access-application-cluster/web-ui-dashboard.md

```diff
@@ -57,7 +57,27 @@ kubectl proxy
 
 Kubectl will make Dashboard available at http://localhost:8001/api/v1/namespaces/kubernetes-dashboard/services/https:kubernetes-dashboard:/proxy/.
 
-The UI can _only_ be accessed from the machine where the command is executed. See `kubectl proxy --help` for more options.
+<!-- The above statement is implicit about accessing it local , so commenting and removing the below line.
+The UI can _only_ be accessed from the machine where the command is executed. 
+-->
+
+
+To make kubernetes dashboard accessible remote using SSH tunnel and configure proxy:
```

Comment:
Would you remove the sentence that this commented out?

What do you think about this change?
For remote access to a Kubernetes dashboard, set up an SSH tunnel and configure a proxy. For example,

Reply:
The intention is to remove the line.But I commented it out so that we keep the history.Is that a best way or we can remove / replace a line and highlight that in the comment section...

Reply:
`configure` seems to be redundant here.

Reply:
Moreover, we put emphasis on `only` because we still can see the number of people that are trying to access it from remote locations even if their setup doesn't allow that.